### ISSUE TITLE
balena flow from block(s) to fleet(s)

### ISSUE THREAD

**Issue by ab77** [product-os/flowzone#101]
Would we consider supporting a balena workflow that:
(a) [publishes](https://github.com/belodetek/black.box/blob/master/.github/workflows/balena.yml) block(s); and
(b) [deploys](https://github.com/belodetek/black.box/blob/master/.github/workflows/balena.yml#L207-L239) to fleet(s)

I've built this pattern up for a personal project and I feel it works reasonably well.

**Comment by klutchell** [comment]
@ab77 we already have this, you just need to provide the correct `balena_slugs`

Flowzone will call the deploy-to-balena action and neither care if it's an app, fleet, or block. It just pushes to a slug.

**Comment by ab77** [comment]
So once we pushed draft block release, we would need to push a draft release to fleet(s), using the draft block release. So we would just run another flowzone job, with different inputs?

**Comment by klutchell** [comment]
Oh I see, you want to chain the events so a draft block can create a draft app which can create a draft fleet, like that?

I would have to think about that, but it seems outside the scope of a single flowzone call, would probably need multiple with versioning disabled on the 2nd and 3rd calls.

**Comment by ab77** [comment]
> chain the events so a draft block can create a draft app which can create a draft fleet, like that

Yeah, which is the flow I've created in the personal project linked above. There is a bit of logic in there and it may not be as clean as it could be..
